app/views/shared/_recent_activity.html.erb

```diff
@@ -7,6 +7,32 @@
 </h2>
 
 <ul class="recent_activity unstyled_list">
+
+	<% if @current_user && @current_user.last_url %>
```

Comment:
On Thu, Jul 02, 2015 at 08:53:58AM -0700, Brian Sadler wrote:
> fyi: there is some weird spacing stuff going on in this section.  E.g. Pick up where you left off and the actual link are spaced like halfway across the screen.

screenshot? On mine the link shows right under the title.